wdl/subworkflows/filter_bam_reads_with_qc_fail_flag.wdl

```diff
@@ -0,0 +1,30 @@
+version 1.0
+
+
+import "../tasks/samtools.wdl"
+
+
+workflow filter_bam_reads_with_qc_fail_flag {
+    input {
+        File flagged_bam
+        Int? qc_fail_flag = 512
+        Resources resources
+        SamtoolsViewParams params = object {
+            exclude: qc_fail_flag,
+            output_bam: true
+        }
+        String? out_path = "qc_fail_flag_filtered.bam"
```

Comment:
Should we call this something like "remove_reads_with_qc_fail_flag" just to make extra sure there is no way to confuse removing reads with flagging reads? I don't feel super strongly about this, just wondering.

Reply:
We should take back the word filter. Marked versus flagged is probably more confusing.

Reply:
so you suggest "marked/flagged (choose one)" == fiddling bam binary flags;
"remove_reads_with_xyz_flag" == actually removing reads? I would like something like this.